samples/nrf9160/modem_shell/README.rst

```diff
@@ -44,10 +44,31 @@ Enable and configure eDRX::
 
    link edrx -e -m --edrx_value 0010 --ptw 0010
 
-Enable and configure PSM::
+Subscribe for modem TAU and sleep notifications, enable and configure PSM::
 
+   link tau --subscribe
+   link msleep --subscribe
    link psm -e --rptau 01100100 --rat 00100010
 
+Disable autoconnect::
+
+   link nmodeauto --disable
+   
+   Reboot the DK and it remains in pwroff functional mode and link is not connected.
+
+Link settings::
+
+   Some of the link subcommand parameters are stored to settings and are persistent between sessions.
+
+Set custom AT commands that are run when going to normal mode::
+   
+   Prerequisites: modem sysmode pwroff.
+
+   link nmodeat --mem1 "at%xbandlock=2,\"100\""
+   link funmode --normal
+
+   Band #3 is locked to be in use during the next attempt to normal mode.
```

Comment:
If this text is not coming from the code, have it as normal text.

Reply:
Yep. Modifying in my next change